mParticle-Apple-SDK/Utils/MPDateFormatter.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  MPDateFormatter.swift
+//  mParticle-Apple-SDK
+//
+//  Created by Brandon Stalnaker on 9/19/24.
+//
+
+import Foundation
+
+@objc public class MPDateFormatter : NSObject {
+    private static var dateFormatterRFC3339: DateFormatter = {
+        let formatter = DateFormatter()
+        let enUSPOSIXLocale = Locale(identifier: "en_US_POSIX")
+        formatter.locale = enUSPOSIXLocale
```

Comment:
```suggestion
        formatter.locale = Locale(identifier: "en_US_POSIX")
```
No need for the extra variable here, and standardizes with the other ones which directly assign the Locale